src/main/java/ru/hogwarts/school/repository/FacultyRepository.java

```diff
@@ -0,0 +1,12 @@
+package ru.hogwarts.school.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.hogwarts.school.entity.Faculty;
+
+import java.util.List;
+
+public interface FacultyRepository extends JpaRepository<Faculty, Long> {
+    List<Faculty> findByColor(String color);
+    List<Faculty>findByNameOrColorIgnoreCase(String name, String color);
```

Comment:
```suggestion
    List<Faculty>findByNameIgnoreCase(OrColorIgnoreCase(String name, String color);
```
регистр игнорируется только для цвета

Reply:
вот из текста задания непонятно, относится ли "игнорируя регистр" с имени и цвету или только к цвету. но тут. как скажешь, применю условие к цвету

Reply:
а, Дима, сорри, это у меня игнор только на цвет стоит. Всё на свои места встало, мне сначала показалось, что ты обратное имеешь ввиду - надо к цвету, а я к обоим параметрам)).